token/2.0/securesubmit.js

```diff
@@ -0,0 +1,632 @@
+﻿/*global $, jQuery*/
+var hps = (function ($) {
+    "use strict";
+
+    var HPS;
+
+    HPS = {
+
+        Tag: "SecureSubmit",
+
+        Urls: {
+            CERT: "https://posgateway.cert.secureexchange.net/Hps.Exchange.PosGateway.Hpf.v1/api/token",
+            PROD: "https://api.heartlandportico.com/SecureSubmit.v1/api/token",
+            iframeCERT: "https://github.com/hps/hps.github.io/tree/master/token/2.0/securesubmit.js",
```

Comment:
@markhagan Shouldn't this just reference `https://hps.github.io/token/2.0/` as the `src` for the iframe?